api/app/tests/integrations/test_pteams.py

```diff
@@ -0,0 +1,185 @@
+from datetime import datetime
+
+import pytest
+from fastapi.testclient import TestClient
+
+from app import models
+from app.main import app
+from app.tests.common import ticket_utils
+from app.tests.medium.constants import (
+    ACTION1,
+    PTEAM1,
+    TOPIC1,
+    USER1,
+)
+from app.tests.medium.utils import (
+    create_service_topicstatus,
+    headers,
+)
+
+client = TestClient(app)
+
+
+@pytest.mark.parametrize(
+    "topic_status, solved_num, unsolved_num",
+    [
+        (models.TopicStatusType.acknowledged, 0, 1),
+        (models.TopicStatusType.scheduled, 0, 1),
+        (models.TopicStatusType.completed, 1, 0),
+    ],
+)
+def test_get_service_tagged_ticket_ids_when_change_the_topic_status_type(
```

Comment:
testにおいてwhenは「この動作を行った時」というニュアンスを含むので、この関数からは「topic status tyepを変更した時」というニュアンスを受けます

以下のようにtciekt statusに応じて変える値がかわる、ということがわかる名前にするか
```suggestion
def test_it_should_return_number_based_on_ticket_status(
```

もしくは関数を分割して、「completedの場合はsolvedとして扱われることを確かめるテスト」「acknowledged, scheduledの場合はunsolvedとして扱われることを確認するテスト」に分けるのがいいと思います
この二つを分離すると `if solved_num == 1:` のような分岐も不要になるはず